### PR TITLE
Added Starter README + Changed to Apache 2.0 License

### LINKED ISSUE #10: [Packaging] - License and CI/CD
- [ ] Get familiarised with Python + Jupyter packaging
- [ ] Get familiarised with GitHub actions 
   - [ ] Create a CI/CD script that tests the main branch builds correctly 
   - Have in mind that you'd need to test the Python package on the one hand, as well as the Node/React package too.

### PR DESCRIPTION
**Pull request by mrzengel**
Closes #10 #8

### CODE REVIEW

**Review by garciagenrique — CHANGES_REQUESTED** [review]

**Comment by garciagenrique** [comment]
Otherwise LGTM!

File: README.md [hunk 1]
```diff
@@ -1,2 +1,30 @@
-# zenodo-jupyterlab-extension
+# Zenodo Jupyterlab Extension
 Zenodo JupyterLab plugin
+**This project is currently in development and is subject to change.**
+
+This project integrates [Zenodo](https://zenodo.org) into Jupyter Lab extension.
+
+# Requirements
+JupyterLab > 4, > 5
```
[the remark this answers is not in the document]
Comment (mrzengel) [reply README.md]: Should be fixed now

File: README.md [hunk 2]
```diff
@@ -1,2 +1,30 @@
-# zenodo-jupyterlab-extension
+# Zenodo Jupyterlab Extension
 Zenodo JupyterLab plugin
+**This project is currently in development and is subject to change.**
+
+This project integrates [Zenodo](https://zenodo.org) into Jupyter Lab extension.
+
+# Requirements
+JupyterLab > 5
```
Comment (garciagenrique) [inline README.md:8]: This was on me. 
The version of jupyterlab must be lower than 5 and larger than 4.

**Review by garciagenrique — CHANGES_REQUESTED** [review]

**Comment by mrzengel** [comment]
Okay should be good now

**Review by garciagenrique — APPROVED** [review]

**Pull request merged**